### PR TITLE
fix failing tests

### PR DESCRIPTION
**Pull request by meldafrawi**
Signed-off-by: Mohamed Eldafrawi <mohamed.eldafrawi@rancher.com>

### CODE REVIEW

File: manager/integration/tests/test_basic.py [hunk 1]
```diff
@@ -2952,12 +2954,12 @@ def test_backup_lock_creation_during_deletion(set_random_backupstore, client, co
     _, b1 = common.find_backup(client, std_volume_name, snap1.name)
 
     write_pod_volume_random_data(core_api, std_pod_name,
-                                 "/data/test2", DATA_SIZE_IN_MB_1)
-
-    backup_volume.backupDelete(name=b1.name)
+                                 "/data/test2", DATA_SIZE_IN_MB_2)
```
Comment (meldafrawi) [inline manager/integration/tests/test_basic.py:2957]: increase written data size, to take more time in deletion.

File: manager/integration/tests/test_basic.py [hunk 2]
```diff
@@ -2952,12 +2954,12 @@ def test_backup_lock_creation_during_deletion(set_random_backupstore, client, co
...
 
     snap2 = create_snapshot(client, std_volume_name)
 
+    backup_volume.backupDelete(name=b1.name)
+
```
Comment (meldafrawi) [inline manager/integration/tests/test_basic.py:2962]: Create second snapshot first, then consequently delete backup 1 and trying to create backup 2

File: manager/integration/tests/test_ha.py [hunk 3]
```diff
@@ -705,7 +705,10 @@ def test_rebuild_with_restoration(set_random_backupstore, client, core_api, volu
     original_volume = client.by_id_volume(original_volume_name)
     snap = create_snapshot(client, original_volume_name)
     original_volume.snapshotBackup(name=snap.name)
-    wait_for_backup_completion(client, original_volume_name, snap.name)
+    wait_for_backup_completion(client,
+                               original_volume_name,
+                               snap.name,
+                               retry_count=600)
```
Comment (meldafrawi) [inline manager/integration/tests/test_ha.py:711]: Increase time wait for backup to complete.

File: manager/integration/tests/test_ha.py [hunk 4]
```diff
@@ -932,7 +935,10 @@ def test_inc_restoration_with_multiple_rebuild_and_expansion(set_random_backupst
     std_volume = client.by_id_volume(std_volume_name)
     snap1 = create_snapshot(client, std_volume_name)
     std_volume.snapshotBackup(name=snap1.name)
-    wait_for_backup_completion(client, std_volume_name, snap1.name)
+    wait_for_backup_completion(client,
+                               std_volume_name,
+                               snap1.name,
+                               retry_count=600)
```
Comment (meldafrawi) [inline manager/integration/tests/test_ha.py:941]: Increase time wait for backup to complete.

File: manager/integration/tests/test_ha.py [hunk 5]
```diff
@@ -1775,7 +1781,10 @@ def test_engine_crash_for_dr_volume(set_random_backupstore, client, core_api, vo
     snap2 = create_snapshot(client, volume_name)
     volume = client.by_id_volume(volume_name)
     volume.snapshotBackup(name=snap2.name)
-    wait_for_backup_completion(client, volume_name, snap2.name)
+    wait_for_backup_completion(client,
```
Comment (meldafrawi) [inline manager/integration/tests/test_ha.py:1784]: Increase time wait for backup to complete.

**Review by khushboo-rancher — APPROVED** [review]
LGTM

**Pull request merged**